### ISSUE TITLE
Distribute functionally according to module

### ISSUE THREAD

**Issue by chris-b-pacon** [pinosante/VAM-Evolutionary-Character-Creation#29]
There is still much functionality in GUI that would belong into business logic and vice versa. Unfortunately the methods inside GUI are very complicated and have to be dissected in a cautious way to split the GUI parts from the logic parts.

**Comment by pinosante** [comment]
Yeah, this is why I gave up on splitting the code. There is some hard choices to be made.

**Comment by chris-b-pacon** [comment]
Don't worry, I am a professional! :D

**Comment by chris-b-pacon** [comment]
Further splitting should be:
- separating appearance selction dialog into module 
- splitting Chromosome into GUi part and BL part
- creating an Appearance class in separate module
- creating a Morph class in module
- may split Script UI from VAM UI -- have figure out the relations first
